pulp_ansible/app/models.py

```diff
@@ -325,3 +327,7 @@ class AnsibleDistribution(Distribution):
 
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
+
+
+def _gen_secret():
```

Comment:
What's this for?

Reply:
I think, a leftover of the redirect content guard. Should be removed.

Reply:
oh woops. I missed that when I removed the content redirect guard.